app/src/main/java/com/example/workoutapp/dto/Workout.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.workoutapp.dto
+
+class Workout {
```

Comment:
I couldn't  find your json link anywhere to come up with the workout class myself